docs/pages/cloud/guides/webauthn.mdx

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
Instead of having a separate cloud guide, you can now use scopes:

https://raw.githubusercontent.com/gravitational/teleport/e1774a8f8ba58f25bc1f7d86657b6d4a972ccb15/docs/pages/database-access/guides/dynamic-registration.mdx

https://raw.githubusercontent.com/gravitational/teleport/2ccd6461afbd3da5e10910edfaa3bc9655bd0b50/docs/pages/enterprise/workflow/ssh-approval-slack.mdx

Locking article refresh example:

https://github.com/gravitational/teleport/commit/6dc56d558f7ef7ff777f5cbbe642ebf097d901b4#diff-8508c129e30023dde3e081e66d9fcc835ee304b3582a14fd896b578eedbdf4e5

Please push this content to the original article on webauthn and use scopes to modify content on the fly

Reply:
Done.

I wasn't too sure what to do with it at first, so it seemed best to update it. Comparing both articles now, specially after the added tabs for Cloud/Dynamic config, it looks like we covered everything, so I went and removed the Cloud guide (and the now-empty section with it).